plugins/file-open-save/core/src/main/resources/project.properties

```diff
@@ -0,0 +1 @@
+CLIENT_PATH=/8.0-SNAPSHOT/file-open-save/index.html
```

Comment:
the path is wrong, and this must be filtered. move to `resources-filtered`
`CLIENT_PATH=/${project.artifactId}/${project.version}/index.html`